components-rs/telemetry.rs

```diff
@@ -166,3 +171,24 @@ pub unsafe extern "C" fn ddog_sidecar_telemetry_add_span_metric_point_buffer(
         tags,
     )));
 }
+
+#[no_mangle]
+pub unsafe extern "C" fn ddog_sidecar_telemetry_add_integration_log_buffer(
+    buffer: &mut SidecarActionsBuffer,
+    log: CharSlice
+) {
+    let mut hasher = DefaultHasher::new();
```

Comment:
I think a stable hash would be better here? We use ZwoHasher for that purpose in libdatadog.

Reply:
I did that because this is what is used [here](https://github.com/DataDog/libdatadog/blob/0957884d15dac4aa6df77cd258aea4fd192532d2/ddtelemetry/src/worker/mod.rs#L848). But i can switch to ZwoHasher